README.md

```diff
@@ -140,7 +175,13 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+GET - shows an instance
+show - shows an instance
+new - creates a new instance (needs to be saved)
+create - creates a new instance that saves automatically
+edit - edits instance (needs to be saved)
+update - edits an instance that saves automatically
+destroy - deletes and instance
 ```
```

Comment:
You demonstrate this understanding with your answer, but it's worth saying explicitly: a RESTful route consists of a method/verb and a path.